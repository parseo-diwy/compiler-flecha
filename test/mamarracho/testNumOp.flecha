def sum = 1 + 2
def sub = 43 - 1
def mul = 3 * 3
def div = 16 / 8
def main =
  unsafePrintInt sum; unsafePrintChar '\n';
  unsafePrintInt sub; unsafePrintChar '\n';
  unsafePrintInt mul; unsafePrintChar '\n';
  unsafePrintInt div
