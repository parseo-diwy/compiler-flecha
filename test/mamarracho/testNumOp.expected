3
42
9
2